ocs_ci/helpers/managed_services.py

```diff
@@ -22,6 +22,74 @@
 log = logging.getLogger(__name__)
 
 
+def get_machine_pool_info_list():
```

Comment:
Can this be moved to ocs_ci/utility/rosa.py? It looks like a reusable function that could improve rosa module.

Reply:
Yes, sounds good.